src/java.base/share/classes/java/util/zip/ZipCoder.java

```diff
@@ -56,28 +56,27 @@ public static ZipCoder get(Charset charset) {
     }
 
     /**
-     * This enum represents the three possible return values for
+     * These values represents the three possible return values for
```

Comment:
Consider simplifying to

```suggestion
     * Constants representing the three possible return values for
```